science/gwyddion/Portfile

```diff
@@ -40,7 +40,7 @@ configure.args      --disable-desktop-file-update \
                     --x-includes=${prefix}/include \
                     --x-libraries=${prefix}/lib
 
-patchfiles          patch-libgwyddion-gwyutils-c.diff
+# patchfiles          patch-libgwyddion-gwyutils-c.diff
```

Comment:
If the patchfile isn't needed any more, could we remove it (and this line as well?)

Reply:
done